docs/en/stack/security/authentication/built-in-users.asciidoc

```diff
@@ -20,12 +20,11 @@ storing monitoring information in {es}. It has the `remote_monitoring_agent` and
 [float]
 [[built-in-user-explanation]]
 ==== How the built-in users work
-These built-in users are stored within a special `.security` index managed by
-{security}.
-This means that, if the password is changed, or a user is disabled, then that
-change is automatically reflected on each node in the cluster. It also means
-that if your `.security` index is deleted, or restored from a snapshot, then
-any changes you have applied will be lost.
+These built-in users are stored in a special `.security` index, which is managed
+by the {security-features}. If a built-in user is disabled or its password
```

Comment:
I'm OK with this, but I think it would read better to say
```suggestion
by {es}. If a built-in user is disabled or its password
```
or even `{es-security-features}`. It just feels weird to me, to suggest that a _feature_ manages something.